nh_neurological/models/nh_clinical_patient_observation_neurological.py

```diff
@@ -35,8 +35,6 @@ class NhClinicalPatientObservationNeurological(models.Model):
     ]
 
     _description = "Neurological"
-    # TODO Remove when EOBS-982 complete.
```

Comment:
Did you mean to delete this TODO? Tech debt EOBS-982 is still open and the code that it refers to is still in the file. Also, it might be you want to add this TODO or JIRA reference to the other files that need changing too, will make it a lot easier to find when we get to make the changes.